src/shared/capture/capturev4l.cpp

```diff
@@ -581,7 +577,8 @@ CaptureV4L::CaptureV4L(VarList * _settings,int default_camera_id) : CaptureInter
     }
     
     if (cam_id > cam_list[cam_count-1]) {
-        fprintf(stderr,"CaptureV4L Error: no camera found with index %d. Max index is: %d\n",cam_id,cam_list[cam_count-1]);
+        //fprintf(stderr,"CaptureV4L Error: no camera found with index %d. Max index is: %d\n",cam_id,cam_list[cam_count-1]);   /// 2/5/16 --- shh, please be quiet
```

Comment:
I'm a bit uncomfortable about silently gobbling up an error message like this, which might actually help the user. The DC1394 driver explicitly prints an error when asked to open a camera Idx > num_cameras - 1. 
How about a printonce-like functionality as a compromise?
{
  static bool printed_once = false;
  if (!printed_once) {
    fprintf(etderr, "Error... This message will print only once.");
    printed_once = true;
  }
}